docs/reference/search/search-your-data/knn-search.asciidoc

```diff
@@ -354,6 +321,11 @@ each score in the sum. In the example above, the scores will be calculated as
 score = 0.9 * match_score + 0.1 * knn_score
 ```
 
+The `knn` option can also be used with <<search-aggregations, `aggregations`>>. In general, {es} computes aggregations
+over all documents that match the search. So for approximate kNN search, aggregations are calculated on the top `k`
+nearest documents. If the search also includes a `query`, then aggregations are calculated on the combined set of `knn`
+and `query` matches.
+
```

Comment:
I guess it is obvious here, that if a `query` is not specifically provided aggregations are calculated only on the top `k` from knn query. Or do you think we should all add this clarification, as this is different how aggs would work without knn section?

Reply:
I tried to explain that in this sentence: "So for approximate kNN search, aggregations are calculated on the top `k` nearest documents." Is there something we could add to clarify this?